fixtures/product-by-handle-fixture.js

```diff
@@ -127,6 +135,14 @@ export default {
               "available": true,
               "weight": 0,
               "price": "788.00",
+              "priceV2": {
+                "amount": "0.00",
```

Comment:
It's a fixture, so it doesn't really matter that the priceV2 is being set to 0 here, but just calling out that the variant price is 788.

Reply:
Fixed!